.github/workflows/continuous.yml

```diff
@@ -201,18 +201,35 @@ jobs:
         path: RACK/rack-box/files
         key: files-${{ github.sha }}
 
-    - name: Build rack-box Docker image
+    - name: Build rack-box dev image
+      if: github.ref == 'refs/heads/master'
       run: |
         cd RACK/rack-box
         packer build -var version=dev rack-box-docker.json
 
+    - name: Build rack-box dev-test image
+      if: github.ref == 'refs/heads/dev-test'
+      run: |
+        cd RACK/rack-box
+        packer build -var version=dev-test rack-box-docker.json
+
+    - name: Build rack-box other image
+      if: github.ref != 'refs/heads/master' && github.ref != 'refs/heads/dev-test'
+      run: |
+        cd RACK/rack-box
+        packer build -var version=other rack-box-docker.json
```

Comment:
My motivation for writing this step is to allow any feature or fix branch with a different name than "master" or "dev-test" to test that it can build a rack-box Docker image successfully, even though we push only the "dev" and "dev-test" Docker images to Docker Hub in later steps.

I'm not sure if you want to retain the dev-test branch for future work.  Anytime you or someone else want to continue working on the dev-test branch after some commits have been pushed to the master branch, you will have to remember that you must run `git rebase origin/master` to bring the dev-test branch up to date with the master branch before you append any new commits to the dev-test branch.  You may want to remove the dev-test branch and simply recreate the dev-test branch from the master branch whenever you need to use it.  Note if you decide you will never use the dev-test branch again, then you don't need to change this workflow (all of the extra lines or extra steps or changed lines were made to support the dev-test branch).

Reply:
That's right, the if statements should not break if the dev-test branch is missing.